code_samples/paging_predicate_sample/main/src/main/java/Comparator.java

```diff
@@ -0,0 +1,47 @@
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
+
+import java.io.IOException;
+import java.util.Map;
+
+public class Comparator implements java.util.Comparator<Map.Entry<String, Double>>, IdentifiedDataSerializable {
```

Comment:
We have the same files above right? What is the point?

Reply:
yeah we can remove them